### PR TITLE
feat: add withdraw

### LINKED ISSUE #199: Add withdraw MENTO functionality
**Description**
Atm it is not possible to withdraw MENTO that is already unlocked from existing lock in the UI (the only option to do that is to call the withdraw function of the smart contract directly).
This issue aims to add withdraw functionality to "My voting power" screen.
The text of the button should be dynamic and include the amount of MENTO user can withdraw. If user has multiple locks, the same button will repeat multiple times.

If user has 0 MENTO to withdraw don't show the button.

both desktop and mobile[ designs in Figma](https://www.figma.com/design/QSQsvksBOspBTONiBBFpqa/MENTO-Airdrop-%26-Governance-UIs?node-id=4347-2) are updated to reflect this change.

**Acceptance criteria**

- [x] It is possible to withdraw unlocked MENTO from the "My voting power" screen of the governance app

### PR DESCRIPTION
**Pull request by bayological**
### Description

Updates My Voting Page to include a button to withdraw all unlocked Mento.

### Other changes

- None

### Tested

- Checked mobile view and did [withdrawals](https://alfajores.celoscan.io/tx/0x64f3c7b71d94cad81a4531d46e20434048f59a49b0bb6c88184cbff4305c8f17) on Alfajores.

### Related issues

- Fixes #199

### CODE REVIEW

**Review by philbow61 — APPROVED** [review]
LGTM 👍 
Withdrew tokens from my personal lock on mainnet. 
Worked smooth 💯

File: src/app/my-voting-power/page.tsx [hunk 1]
```diff
@@ -3,10 +3,60 @@ import { useAccount } from "wagmi";
...
+import { useAvailableToWithdraw } from "@/lib/contracts/locking/useAvailableToWithdraw";
+import { useWithdraw } from "@/lib/contracts/locking/useWithdraw";
+import React from "react";
+import { TxModal } from "@/components/_shared/tx-modal/tx-modal.component";
+import { toast } from "sonner";
 
 const Page = () => {
   const { address, isConnecting } = useAccount();
   const { locks, refetch } = useLocksByAccount({ account: address! });
+  const { availableToWithdraw, refetchAvailableToWithdraw } =
```
Comment (denviljclarke) [inline src/app/my-voting-power/page.tsx:15]: You could put all of this logic in a custom useWithdrawMento hook and use that hook in withdraw button. The voting power page shouldn't care about any of this logic & would clean up all of these props

Comment (bayological) [reply src/app/my-voting-power/page.tsx:15]: Good suggestion, will definitely make this cleaner.

**Review by denviljclarke — APPROVED** [review]
nice once, LGTM 🫡

**Pull request merged**